app/assets/javascripts/discourse/lib/desktop-notifications.js.es6

```diff
@@ -55,6 +55,24 @@ function init(messageBus, appEvents) {
   }
 }
 
+function confirmNotification() {
+  const notification = new Notification(I18n.t("notifications.popup.confirm_title", {site_title: Discourse.SiteSettings.title}), {
+    body: I18n.t("notifications.popup.confirm_body"),
+    icon: Discourse.SiteSettings.logo_small_url || Discourse.SiteSettings.logo_url,
+    tag: "confirm-subscription"
+  });
+
+  function clickEventHandler() {
```

Comment:
I know in the past we had some issues with functions declared by name as closures. It might not be an issue any more but it's probably safer to say: `const clickEventHandler = () => notification.close;`